mifosng-provider/src/main/resources/sql/migrations/core_db/V206_8__password_validation_policy_permissions.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO m_permission (grouping, code, entity_name, action_name, can_maker_checker)
+VALUE ("authorisation","READ_PASSWORD_VALIDATION_POLICY","PASSWORD_VALIDATION_POLICY","READ",0);
+
+INSERT INTO m_permission (grouping, code, entity_name, action_name, can_maker_checker)
+VALUE ("authorisation","UPDATE_PASSWORD_VALIDATION_POLICY","PASSWORD_VALIDATION_POLICY","UPDATE",0);
+
```

Comment:
not sure if we need maker checker for this functionality, but better to add the same I guess (done while merging this pr)